plugin.php

```diff
@@ -1,6 +1,12 @@
 <?php
 /**
- * Cavalcade!
+ * Plugin Name: Cavalcade
+ * Plugin URI: https://github.com/humanmade/Cavalcade
+ * Description: Scalable job system, designed as a drop-in replacement for wp_cron
+ * Version: 0.1
```

Comment:
Unless it's required, I'd rather leave off the version until we start versioning Cavalcade.